src/stdatamodels/jwst/datamodels/tests/test_models.py

```diff
@@ -142,7 +142,7 @@ class _NonstandardPrimaryArrayModel(JwstDataModel):
         def get_primary_array_name(self):
             return "wavelength"
 
-    m = _NonstandardPrimaryArrayModel((10,))
+    m = _NonstandardPrimaryArrayModel((10, 10))
```

Comment:
This appeared to be an instance of an array passing validation that we would expect to fail, and indeed it does fail as of the bugfix in this PR.  It doesn't look like this change makes much difference for what the test intends to cover, but someone can correct me if that's wrong.